rpcs3/Emu/System.h

```diff
@@ -147,6 +147,15 @@ enum class frame_limit_type
 	_auto,
 };
 
+enum class perf_overlay_detail_level
+{
+	minimal_1,	// fps only
```

Comment:
why add number suffix to the enum members

Reply:
Because the names could be considered subjective, it's clearer this way.
I don't feel strongly about this, should I remove it?